src/ctapipe/image/muon/ring_fitter.py

```diff
@@ -32,22 +51,44 @@ class MuonRingFitter(Component):
 
     fit_method = traits.CaselessStrEnum(
         list(FIT_METHOD_BY_NAME.keys()),
-        default_value=list(FIT_METHOD_BY_NAME.keys())[0],
+        default_value="kundu_chaudhuri_taubin",
     ).tag(config=True)
 
-    def __call__(self, fov_lon, fov_lat, img, mask):
-        """allows any fit to be called in form of
-        MuonRingFitter(fit_method = "name of the fit")
+    def __call__(self, geom, image, clean_mask):
+        """
+        Parameters
+        -----------
+        geom: CameraGeometry
+        Defines the pixel coordinates
+        image: np.ndarray[np.float]
+        Image intensity values
+        clean_mask: np.ndarray[np.bool]
+        Boolean mask of clean pixels, where True means the pixel contains signal.
+        This can be generated using a `ctapipe.image.ImageCleaner`.
+
+        Returns
+        --------
+        MuonRingContainers:
```

Comment:
Type (my fault): `MuonRingContiners`  should be `MuonRingContainer` (it only returns one, and this allows Sphinx to properly link it)